content/en/docs/concepts/configuration/resource-bin-packing.md

```diff
@@ -0,0 +1,197 @@
+---
+reviewers:
+- bsalamat
+- k82cn
+- ahg-g
+title: Resource Bin Packing for Extended Resources
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="1.16" state="alpha" >}}
+
+The kube-scheduler can be configured to enable bin packing of resources along with extended resources using `RequestedToCapacityRatioResourceAllocation` priority function. Priority functions can be used to fine-tune the kube-scheduler as per custom needs. 
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Enabling Bin Packing using RequestedToCapacityRatioResourceAllocation Priority Function
```

Comment:
is it possible to re-word this to a shorter sentence?

Reply:
I am thinking along ```Enabling Bin Packing using RequestedToCapacityRatioResourceAllocation```

Reply:
👍 I'll update that